packages/authenticator/amplify_authenticator/lib/src/l10n/auth_strings_resolver.dart

```diff
@@ -25,21 +21,27 @@ class AuthStringResolver {
   /// {@macro amplify_authenticator.auth_string_resolver}
   const AuthStringResolver({
     ButtonResolver? buttons,
-    CountryResolver? countries,
+    // ignore: deprecated_member_use_from_same_package
+    @Deprecated('Use areas instead') CountryResolver? countries,
+    DialCodeResolver? areas,
```

Comment:
I don't think `areas` is correct. See https://github.com/aws-amplify/amplify-flutter/pull/3354#discussion_r1267128723.